hack/testing/prep-host

```diff
@@ -7,7 +7,7 @@ ANSIBLE_RPM=${ANSIBLE_RPM:-ansible}
 RUAMEL_YAML_RPM=${RUAMEL_YAML_RPM:-python-ruamel-yaml}
 
 OS_ANSIBLE_REPO=${OS_ANSIBLE_REPO:-https://github.com/openshift/openshift-ansible}
-OS_ANSIBLE_BRANCH=${OS_ANSIBLE_BRANCH:-master}
+OS_ANSIBLE_BRANCH=release-1.5
```

Comment:
the only issue with allowing you to set the branch is that we are using the jenkins job on master so OS_ANSIBLE_BRANCH will always be populated with master